### DISCUSSION TITLE
I keep getting denied for Github Education

### DISCUSSION THREAD

**Discussion by angelospap7** [community/community#68725] [GitHub Education]
### Hi everyone,

I am trying to get approved for Github education. It has expired and now I am putting a form that STATES that I am an OFFICIAL SCHOOL STUDENT and I keep getting the same error: 
You are unlikely to be verified until you have completed your GitHub user profile with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.
Have you completed your GitHub user profile with all your relevant information, such as your full name as it appears in your image and a short bio?
I have done all these, I have changed my name as it states on the paper. And I saw that we can't send support tickets for this. I go to a greek school so my form is in Greek. Is that the problem? Plus, they only let me take a photo, the upload option is gone.
Please help me!!!! Thank you in advance!!

**Comment by Abdullahsaleh203** [comment]
hello

**Comment by Satyxm** [comment]
Hey!

I think the right thing to do is to contact GitHub support through social media with your verification issue or send an email detailing the problem.

I hope that your problem will be solved!

**Reply by angelospap7** [reply]
Hi, when I went on the form on GitHub support, it says that they don't look at each form individually, so that's why I sent a question here. And also, do they respond on social media?

**Comment by samus-aran** [comment]
Hi @angelospap7 👋 

Looking at your profile I suspect it's a matter of translation. Althought the system is pretty robust it doesn't capture all languages. Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍

**Reply by angelospap7** [reply]
So, for the first thing that you said, I tried creating an ocr translation and because I could only insert one image, I put both images in one picture and on the translation, I put the word translated on it and then it said that it's edited and now the option to upload an image is locked. So, how can I put the english translation next to it, then? Because when I try to take a photo from my iPhone, it only lets me use the front camera, so it's a little hard to take a picture. 
As for the second thing that you said, my username is not exactly my name, but I have changed my profile name to exactly as it says on the form that states that I go to school. What can I do? Thank you in advance.

**Reply by samus-aran** [reply]
> So, for the first thing that you said, I tried creating an ocr translation and because I could only insert one image, I put both images in one picture and on the translation, I put the word translated on it and then it said that it's edited and now the option to upload an image is locked. So, how can I put the english translation next to it, then? Because when I try to take a photo from my iPhone, it only lets me use the front camera, so it's a little hard to take a picture.

We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
3. If you can, print off your evidence and take a photo of this copy.
4. Using a phone lie the printed evidence on a flat surface (rather than holding it using one hand...)
5. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.
6. If you are needing to translate then copy the original evidence and lay next the translated version on a separate piece of paper

We have raised the camera (front vs back) to our internal team to investigate 👍

> As for the second thing that you said, my username is not exactly my name, but I have changed my profile name to exactly as it says on the form that states that I go to school. What can I do? Thank you in advance.

I suspect the language you are using is failing to be translated. We would still recommend you translate your name (and important pieces of your evidence) into English if possible using the above tips.

Hope this helps.

**Reply by angelospap7** [reply]
I have changed my name in greek exactly as it states on the paper that says that I go to school, but I can't write my username in greek, so should I just leave it? 

> Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. 

The problem is that I take the photo and then the resolution is becoming worse.

> If you can, print off your evidence and take a photo of this copy.

It is printed.

> If you are needing to translate then copy the original evidence and lay next the translated version on a separate piece of paper

Should I OCR the form and translate it and then print it and put it next to the original form, and maybe write with a pen on it "TRANSLATED"?

**Reply by dnthang** [reply]
This actually worked !!!!!

**Reply by Airaat** [reply]
Hello friend! Did you solve your problem?

**Comment by supuna97** [comment]
hey!! Contact GitHub supporters. That is an easy way to solve this problem.

**Reply by angelospap7** [reply]
Hi, when I went on the form on GitHub support, it says that they don't look at each form individually, so that's why I sent a question here. So do you think that if I send them there, they will respond?

**Reply by supuna97** [reply]
yeah. Did u get the ticket?

**Reply by angelospap7** [reply]
no, because it doesn't even let me open a ticket.

**Comment by JakobHavtorn** [comment]
Doesn't work for me either. I've tried 20+ times with different methods (uploading scans, taking photos, having translations next to the ID, translating all or parts only...). But I imagine it works really well with US student IDs 🙄

**Reply by angelospap7** [reply]
Oh, that's not good at all

**Comment by angelospap7** [comment]
unfortunately nope..

**Reply by Airaat** [reply]
My friend, I think I have found the solution! Try taking a photo from your phone, even if you have to take it with the front camera. Just received "approved"

**Comment by BilalBoudjema** [comment]
# Guide to Resolving GitHub Education Approval Issues

Experiencing difficulties in getting approved for GitHub Education can be challenging. Below are steps and considerations to help you successfully navigate this process.

## Steps to Resolve Approval Issues

### 1. Ensure Profile Completeness
- Your GitHub profile should have:
  - Your full name as it appears in your academic documents.
  - A clear and concise bio.
  - Profile visibility set to public.

### 2. Address Language of Documentation
- If your documents are in Greek, consider adding an English translation alongside the original.

### 3. Photograph Quality of Documents
- Make sure the photo of your academic document is clear and legible.

### 4. Re-apply After Making Changes
- Log out and then log back into GitHub before re-applying for the Education Pack.

### 5. Reach Out to GitHub Support
- Contact GitHub Support explaining your situation, even if it seems that support tickets for this issue are not typical.

### 6. Review GitHub Education FAQs
- Check the [GitHub Education FAQs](https://education.github.com/faq) for specific guidance.

### 7. Use University Email Address
- Link an email address provided by your educational institution to your GitHub account.

### 8. Address Document Upload Issues
- If you're unable to upload documents, mention this in your communication with GitHub Support.

### 9. Explore Community Forums
- Look for advice on GitHub’s community forums where others might have shared solutions for similar issues.

### 10. Regularly Check for Updates
- Keep an eye on your email for any responses from GitHub.

## Additional Notes

- Approval processes can take time, so patience is key. If all guidelines are followed and issues persist, GitHub Support is your best resource for a resolution.

**Reply by eneskutlay** [reply]
In fact, these guidelines lead to no real conclusions. GitHub refused to accept my student document, which I had previously submitted and gotten approved, four times in 24 hours, all rejected, with no explanation. It says my official student document looks like a letter of intent. It's ridiculous.
